src/assets/scripts/client/ui/UiController.js

```diff
@@ -156,6 +156,16 @@ class UiController {
          */
         this.$log = null;
 
+        /**
+         * Duration for chat log transmissions to remain visible
```

Comment:
What is the time value: Miliseconds or Seconds? We should add that in the doc block here for future us 😉